src/AstrometryFit.cc

```diff
@@ -504,25 +362,22 @@ void AstrometryFit::accumulateStatImage(ImType &image, Accum &accu) const {
         Eigen::Vector2d res(fittedStarInTP.x - outPos.x, fittedStarInTP.y - outPos.y);
         double chi2Val = res.transpose() * transW * res;
 
-        accu.addEntry(chi2Val, 2, ms);
+        accum.addEntry(chi2Val, 2, ms);
     }  // end of loop on measurements
 }
 
-//! for a list of images.
-template <class ListType, class Accum>
-void AstrometryFit::accumulateStatImageList(ListType &list, Accum &accum) const {
-    for (auto &im : list) {
-        accumulateStatImage(*im, accum);
+void AstrometryFit::accumulateStatImageList(CcdImageList const &ccdImageList, Chi2Accumulator &accum) const {
+    for (auto &ccdImage : ccdImageList) {
```

Comment:
Would prefer `auto const &` here, since the iteration variable is in fact const.